src/bundle/Resources/public/js/alloyeditor/src/base/ez-blocktextalign.js

```diff
@@ -59,10 +59,9 @@ export default class EzBlockTextAlign extends Component {
         const icon = '/bundles/ezplatformadminui/img/ez-icons.svg#' + this.props.iconName;
 
         return (
-            <button className={cssClass} onClick={this.applyStyle.bind(this)}
-                tabIndex={this.props.tabIndex} title={this.props.label}>
-                <svg className='ez-icon ez-btn-ae__icon'>
-                    <use xlinkHref={icon}></use>
+            <button className={cssClass} onClick={this.applyStyle.bind(this)} tabIndex={this.props.tabIndex} title={this.props.label}>
+                <svg className="ez-icon ez-icon--small ez-icon--light ez-btn-ae__icon">
```

Comment:
I think there is no point to add the `ez-icon-light` class. We are about to change the styling of AE and we will have to remove this class.